src/System.ServiceModel.NetTcp/ref/System.ServiceModel.NetTcp.cs

```diff
@@ -51,6 +51,8 @@ public sealed partial class TcpTransportSecurity
         public TcpTransportSecurity() { }
         [System.ComponentModel.DefaultValueAttribute((System.ServiceModel.TcpClientCredentialType)(1))]
         public System.ServiceModel.TcpClientCredentialType ClientCredentialType { get { return default(System.ServiceModel.TcpClientCredentialType); } set { } }
+        [System.ComponentModel.DefaultValueAttribute((System.Security.Authentication.SslProtocols)(4080))]
```

Comment:
Enum value is `SslProtocols.Ssl3 | SslProtocols.Tls | SslProtocols.Tls11 | SslProtocols.Tls12`
However, the tool that generates this file uses integer values